client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -110,6 +258,18 @@
 			{#if selectedPiece}
 				<div></div>
 			{/if}
+			{#if orders}
+				<div></div>
+			{/if}
 		{/if}
 	</span>
 </div>
+<div class="flex">
```

Comment:
The net result of breaking out the components would be that an  `Arena` has a clear purpose, which is to render an `ArenaCanvas`, an `ArenaLegend` and an `ArenaToolTip`.  Then each of those components can have their own focused purpose.

The `store` from svelte might be the missing link, if the problem is that passing data around becomes a disaster.